python/json_util.py

```diff
@@ -0,0 +1,56 @@
+import mgp
+import json
+from urllib.request import Request, urlopen
+from urllib.error import URLError
+from pathlib import Path
+
+
+def extract_objects(file):
+    objects = json.load(file)
+    if type(objects) is dict:
+        objects = [objects]
+    return objects
+
+
+@mgp.read_proc
+def load_from_path(ctx: mgp.ProcCtx, path: str) -> mgp.Record(objects=mgp.List[object]):
+    """
+    Procedure to load JSON from a local file. 
+
+    Parameters
+    ----------
+    path : str
+        Path to the JSON that is being loaded.
+    """
+    file = Path(path)
+    if file.exists():
+        opened_file = open(file)
+        objects = extract_objects(opened_file)
+    else:
+        raise FileNotFoundError("There is no file " + path)
+
+    opened_file.close()
+
+    return mgp.Record(objects=objects)
+
+
+@mgp.read_proc
+def load_from_url(ctx: mgp.ProcCtx, url: str) -> mgp.Record(objects=mgp.List[object]):
+    """
+    Procedure to load JSON from a remote address. 
+
+    Parameters
+    ----------
+    url : str
+        URL to the JSON that is being loaded.
+    """
+    request = Request(url)
+    request.add_header('User-Agent', 'MAGE module')
+    try:
+        content = urlopen(request)
+    except URLError:
+        print("There is no file " + url)
```

Comment:
We should raise an exception rather than print and ignore

Reply:
I added it, not sure if that's what you wanted, but I tested the wrong URL, and get
`urllib.error.HTTPError: HTTP Error 404: Not Found`
so I hope that's fine

Reply:
We should catch any kind of error that can occur there, raising it again will result with a trace written in console log